cifsd/netlink.c

```diff
@@ -247,46 +247,6 @@ int cifsd_nl_exit(void)
 	return 0;
 }
 
-static void termination_handler(int signum)
-{
-	int err = 0;
-	struct cifsd_client_info *client;
-
-	failed_connection = 0;
```

Comment:
failed_connection and connection global variable are needed anymore ?
If it is no longer used, let's remove all.
I can see declare and codes in pipecb.c and netlink.h.